docs/examples/hubcap.owl

```diff
@@ -0,0 +1,85 @@
+<?xml version="1.0"?>
+<rdf:RDF xmlns="http://www.semanticweb.org/vinicius/ontologies/2024/7/untitled-ontology-2/"
+     xml:base="http://www.semanticweb.org/vinicius/ontologies/2024/7/untitled-ontology-2/"
+     xmlns:owl="http://www.w3.org/2002/07/owl#"
+     xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
+     xmlns:xml="http://www.w3.org/XML/1998/namespace"
+     xmlns:xsd="http://www.w3.org/2001/XMLSchema#"
+     xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#">
+    <owl:Ontology rdf:about="http://www.semanticweb.org/vinicius/ontologies/2024/7/untitled-ontology-2"/>
+    
+
+
+    <!-- 
+    ///////////////////////////////////////////////////////////////////////////////////////
+    //
+    // Object Properties
+    //
+    ///////////////////////////////////////////////////////////////////////////////////////
+     -->
+
+    
+
+
+    <!-- http://www.semanticweb.org/vinicius/ontologies/2024/7/untitled-ontology-2#has_part -->
```

Comment:
These are very ugly IRIs, maybe run a string replace on the file to replace

```
http://www.semanticweb.org/vinicius/ontologies/2024/7/untitled-ontology-2
```

with something like

```
http://example.org/ontology
```